src/blockchain/new-block-emitters.ts

```diff
@@ -127,3 +130,20 @@ export class ListeningNewBlockEmitter extends AutoStartStopEventEmitter {
     this.subscription?.unsubscribe(error => { this.logger.error(error) })
   }
 }
+
+export class NewBlockEmitterService implements Partial<ServiceMethods<any>> {
```

Comment:
Hmm can you call this service with GET request? To get the latest block number? I think there should be `get()` method where you return the `getLastBlockNumber` no?

Reply:
Hmm, as I understand `get`  method require `id` argument, `find` method should be just GET request.

Reply:
Yeah you are right. But still there is no find here right? 😅

Reply:
yes=). So which method should be the best to fetch last block?

Reply:
Well `find` I would say :-) Simple `GET` and it should return you something like `{ latestBlockNumber: ... }` I guess?

Reply:
yea, but if you use feather client on another side should you have a small confusion?